code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -241,6 +241,28 @@ func (msc *StorageSmartContract) GetTransactionByHashHandler(
 	return transaction, err
 }
 
+func (msc *StorageSmartContract) GetBlockTotalByBlockRange(ctx context.Context, params url.Values, balances cstate.StateContextI) (interface{}, error) {
+
+	roundFrom, err := strconv.Atoi(params.Get("round_from"))
+	if err != nil {
+		return nil, errors.New("round from not provided")
+	}
+
+	roundTo, err := strconv.Atoi(params.Get("round_to"))
+	if err != nil {
+		return nil, errors.New("round from not provided")
+	}
+
+	total, err := balances.GetEventDB().GetBlocksMintTotal(int64(roundFrom), int64(roundTo))
```

Comment:
For the  moment we should check `balances.GetEventDb() != nil` to avoid a possible panic.